pytplot/tplot_math/add.py

```diff
@@ -4,22 +4,16 @@
 # Verify current version before use at: https://github.com/MAVENSDC/Pytplot
 
 import pytplot
-import pydivide
-import numpy as np
-from scipy import interpolate
-from scipy.interpolate import interp1d
-import pandas as pd
-
 
 #ADD TWO ARRAYS
 #add two tvar data arrays, store in new_tvar
-def add(tvar1,tvar2,new_tvar,interp='linear'):
+def add(tvar1,tvar2,new_tvar='tvar_add',interp='linear'):
```

Comment:
I wouldn't just call it "tvar_add", I would have it involve the other two tvar names that the user is adding.  So the default final name would be something like "diff_e_flux_diff-ion-flux_add" or something if the two variables are named "diff_e_flux" and "diff_ion_flux".  This applies the the rest of the default names.